doc/api/url.md

```diff
@@ -129,9 +129,9 @@ return `true`.
 
 #### `new URL(input[, base])`
 
-* `input` {string} The absolute or relative input URL to parse. If `input`
+* `input` {any} The absolute or relative input URL to parse. If `input`
```

Comment:
I think I would rather keep `{string}` here, as it's more useful and also more consistent with other web APIs in Node.js.

Reply:
```suggestion
* `input` {string} The absolute or relative input URL to parse. If `input`
```

Reply:
OK, done. Thanks.